deployments/examples/README.md

```diff
@@ -45,6 +45,9 @@ Deployment files for the driver and executor:
 * [driver](spark30/driver.yaml).
 * [executor](spark30/executor.yaml).
 
+## tfjob
```

Comment:
Please use the full name instead of abbreviation: tfjob -> TensorFlow job, or Kubeflow